File: java/vowelsinstring.java
import java.util.Scanner;

public class vowelsinstring {
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a String:");
        String str = sc.nextLine();

        char[] vowels = {'a','e','i','o','u'};
        int inc = 0;

        for(int i=0; i<str.length();i++){
            char c = Character.toLowerCase(str.charAt(i));
            for(char v : vowels){
                if(c == v){
                    inc++;
                }
            }
        }
        System.out.println("Number of vowels in the string: " + inc);
        sc.close();
    }
}

File: C/vowels_in_string.c
#include <stdio.h>
int main() {
    char str[100];
    char vowels[] = {'a', 'e', 'i', 'o', 'u'};
    int inc = 0;

    printf("Enter a String: ");
    fgets(str, sizeof(str), stdin);

    for (int i = 0; str[i] != '\0'; i++) {
        char c = str[i];

        if (c >= 'A' && c <= 'Z') {
            c = c + 32;
        }

        for (int j = 0; j < 5; j++) {
            if (c == vowels[j]) {
                inc++;
            }
        }
    }
    printf("Number of vowels: %d\n", inc);
    return 0;
}